server/plugins/stencil-editor/api/getVariations.js

```diff
@@ -68,7 +68,7 @@ module.exports = function (options, themeConfig) {
                 mobileScreenshot: mobileScreenshot,
                 demoUrl: variation.meta.demo_url,
                 documentationUrl: "string",
-                displayVersion: "string",
+                displayVersion: themeConfig.getVersion(),
```

Comment:
@bc-ong - i've updated it to share a variable.  i'm hesitant to hardcode versionId to something so i kept it as is but it looks like it might not be used.